src/services/checkInService.ts

```
@@ -10,20 +10,19 @@ import {
   deleteField,
 } from "firebase/firestore";
 import { Customer } from "../types";
+import { appConfig } from "../config/config";
 
 // Parse the QR code to extract the ID number
 export const parseQRCode = (qrCode: string): string | null => {
-  // Format: "aux-training-950920-08-6687" or "950920-08-6687"
+  const qrPrefix = appConfig.qrPrefix;
   const parts = qrCode.split("-");
```

ESLint

[error] 'parts' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)